### ISSUE TITLE
SMTP Password Encoding Issue when Connecting to AWS SES

### ISSUE THREAD

**Issue by brunopastorellorocktrade** [mautic/mautic#14146]
### Mautic Series

5.1.x series

### Mautic installed version

5.1.1

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.1

### What browsers are you seeing the problem on?

Not relevant

### What happened?

We have encountered an issue when configuring AWS SES as the SMTP provider in Mautic. The SMTP password (AWS Secret Access Key) is saved encoded in the local.php file, but it seems that Mautic is sending the encoded password during authentication, causing an error when trying to connect to AWS SES.

A temporary workaround is to use an AWS Secret Access Key without special characters, but this is not an ideal solution for long-term security.

### How can we reproduce this issue?

1. Fill in the Email DSN in Settings > Email Configuration.
2. Send a test email.

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by dsp76** [comment]
Interesting catch! Is this related to those issues: https://github.com/mautic/mautic/issues/12256

Looks like this is connected mainly with this: https://github.com/mautic/mautic/issues/13228
Do you agree, that we can close this issue as a duplicate of https://github.com/mautic/mautic/issues/13228?

**Comment by dsp76** [comment]
Closed as duplicate to #13228
